lib/matplotlib/contour.py

```diff
@@ -45,6 +44,35 @@ def get_rotation(self):
         return new_angle
 
 
+def _contour_labeler_handler(cs, inline, inline_spacing, event):
```

Comment:
```suggestion
def _contour_labeler_handler(cs, inline, inline_spacing, event):
    """Event handler for ContourLabeler; used with blocking_input_loop()."""
```
Alternatively name it `_contour_labler_event_handler` but that's quite long and will get you in wrapping troubles below.

Reply:
went for renaming